addons/zylann.hterrain/tools/plugin.gd

```diff
@@ -852,6 +873,60 @@ func _debug_spawn_collider_indicators():
 				mi.position = hit.position
 
 
+
+func _show_brush_size_selector(min_size, max_size, widget_color, initial_value, action_name, on_value_selected):
+	if _widget_action_name != action_name:
+		var active_viewport = _get_active_viewport()
+
+		if active_viewport == null:
+			# when mouse outside 3d viewport area, dont try to show widget
+			return
+
+		_widget_action_name = action_name
+		if active_viewport != null:
+			var selector_scene = ResourceLoader.load("res://addons/zylann.hterrain/tools/brush/brush_size_selector.tscn")
+			var selector = selector_scene.instantiate()
+
+			selector.min_value = min_size
+			selector.max_value = max_size
+			selector.brush_preview_color = widget_color
+
+			_overlay_selector = selector
+			_overlay_selector.position = active_viewport.get_global_mouse_position()
+			_overlay_selector.set_meta("action", action_name)
```

Comment:
Right now, it does nothing. 
This is stored so if there was ever a need to check what the overlay action was about, it is there.
Just in case there were to be more than one type of overlay and they might affect each other.
But you're right. Its better to remove it, and only add if there is a need for it later.